src/translations/en_US.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
   fieldInvalidForPostalCode: 'This postal code is not valid.',
   fieldTooLongForCardholderName: 'Cardholder name must be less than 256 characters.',
   genericError: 'Something went wrong on our end.',
+  hostedFieldsDuplicateCardError: 'This credit card already exists as a saved payment method.',
```

Comment:
I'm open to changing the messaging around this to make it less awkward/technical sounding.

As a side note, I'm not sure if we need to improve our documentation around `failOnDuplicatePaymentMethod` to explicitly state that it's scoped to a merchant's vault and not to a customer ID.

Reply:
I think we do need to update the documentation, it's not particularly clear here: https://developers.braintreepayments.com/reference/request/client-token/generate/node#options.fail_on_duplicate_payment_method